server/channels/app/team.go

```diff
@@ -1503,7 +1503,19 @@ func (a *App) prepareInviteGuestsToChannels(teamID string, guestsInvite *model.G
 		if channel.TeamId != teamID {
 			return nil, nil, nil, model.NewAppError("prepareInviteGuestsToChannels", "api.team.invite_guests.channel_in_invalid_team.app_error", nil, "", http.StatusBadRequest)
 		}
+
+		// Check if the channel has access control policy
```

Comment:
updating the test to cover this started become really complex since it required to create the correct associated parent policy to the channel and starting the PAP (this is where I hit the wall), so I will ask @isacikgoz for guiding on how to get this done and once ready will be pushed in a new PR.